Source/Fuse.Reactive.JavaScript/ObservableProperty.uno

```diff
@@ -8,7 +8,7 @@ namespace Fuse.Reactive
 	{
 		public LazyObservableProperty(ThreadWorker w, Scripting.Object obj, Property p): base(w, obj, p)
 		{
-			w.Context.ObjectDefineProperty(obj, p.Name.ToString(), Get);	
+			w.Context.ObjectDefineProperty(obj, JSName, Get);	
```

Comment:
Or, hm, no, they just point to the same getter. I suppose this is fine. However, I am generally against it because this adds a lot of bloat, makes the syntax uglier and additional members only to allow programmers to not know their own programming language.